packages/hooks/src/stories/UseTransaction.stories.tsx

```diff
@@ -0,0 +1,143 @@
+import { Button, Text, VStack, Input } from '@chakra-ui/react';
+import React from 'react';
+import { NETWORKS, Provider, useWallet, useContract } from '..';
+import { useTransaction } from '../hooks';
+
+export default {
+  title: 'Hooks/useTransaction',
+};
+
+const ADDRESS = '0x7e1D33FcF1C6b6fd301e0B7305dD40E543CF7135'; // Rinkeby
+const ABI = [
+  {
+    inputs: [
+      {
+        internalType: 'string',
+        name: '_greeting',
+        type: 'string',
+      },
+    ],
+    stateMutability: 'nonpayable',
+    type: 'constructor',
+  },
+  {
+    inputs: [],
+    name: 'greet',
+    outputs: [
+      {
+        internalType: 'string',
+        name: '',
+        type: 'string',
+      },
+    ],
+    stateMutability: 'view',
+    type: 'function',
+  },
+  {
+    inputs: [
+      {
+        internalType: 'string',
+        name: '_greeting',
+        type: 'string',
+      },
+    ],
+    name: 'setGreeting',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+  {
+    inputs: [
+      {
+        internalType: 'address payable',
+        name: '_to',
+        type: 'address',
+      },
+    ],
+    name: 'transferTo',
+    outputs: [],
+    stateMutability: 'payable',
+    type: 'function',
+  },
+];
+
+const UsingUseContract = () => {
+  const { connectWallet, disconnectWallet, connected } = useWallet();
+  const contract = useContract(ADDRESS, ABI);
+  const [value, setValue] = React.useState('');
+
+  const [greet, greetLoading, greetError] = useTransaction(async () =>
+    // @ts-expect-error
+    alert(await contract.greet())
+  );
+
+  // @ts-expect-error
```

Comment:
Can you paste the error that you are seeing here?

Reply:
I'm getting this typescript type error `Property 'setGreeting' does not exist on type '{}'.`

![image](https://user-images.githubusercontent.com/56908732/146928571-402c1aab-3eff-4987-840d-6d42d38bbbe0.png)

Reply:
That's expected since the `useContract` does not return any type defs as of now